client/src/resources/lang/en_us.json

```diff
@@ -463,131 +463,131 @@
+    "wiki.article.standard.jail.text": "A Jailor can put a player in jail. When a player is in jail, they can talk with the Jailor at night, the Jailor can choose to execute them, they can't select anyone, and most abilities visiting the jailed player will fail.",
     "wiki.article.standard.rampage.title": "Rampage",
-    "wiki.article.standard.rampage.text": "When all the visitors to a player are attacked by some ability. The player with the ability won't necessarily visit the players being attacked by the rampage.",
+    "wiki.article.standard.rampage.text": "All the visitors to a player are attacked. The player with the ability won't necessarily visit the players being attacked by the rampage.",
     "wiki.article.standard.possess.title": "Possess",
-    "wiki.article.standard.possess.text": "A player who is possessed had their visits changed by the Witch, Necromancer, or Retributionist.",
+    "wiki.article.standard.possess.text": "A player who is possessed had their visits changed by the Witch, Necromancer, or Retributionist. Some roles have immunity to possession.",
     "wiki.article.standard.defense.title": "Defense",
-    "wiki.article.standard.defense.text": "Attack powers are basic, armor piercing, and protection piercing.\n\nDefense powers are armor, protected, and invincible.\n\nSome roles have armor by default, usually evil ones. Defense can be granted to players through abilities like the Doctor's ability to protect a player. A player with invincible defense can only die by getting lynched or leaving the game. A protection piercing attack can overpower armor as well as protection",
+    "wiki.article.standard.defense.text": "Attack powers are either basic, armor-piercing, or protection-piercing.\n\nDefense powers are either armor, protected, or invincible.\n\nA basic attack can not kill a player with armor, and an armor-piercing attack can not kill a player who is protected. A protection-piercing attack can overpower armor as well as protection.\n\n Some roles have armor by default, usually evil ones. Protection can be granted to players through abilities like the Doctor's ability to protect a player. A player with invincible defense can only die by getting lynched or leaving the game.\n\nUnless otherwise indicated, attacks are basic by default and will not be able to kill players who have armor or protection.",
     "wiki.article.standard.protect.title": "Protect",
-    "wiki.article.standard.protect.text": "Usually this means increasing someone's defense to protected for a night, but It could mean many things depending who is protecting who.",
+    "wiki.article.standard.protect.text": "Protect means to increase someone's defense to protected for a night. It may have additional effects depending on the role granting protection.",
```

Comment:
"the role granting protection" You should better explain that one person is protecting another person